ansible/group_vars/proxy-hosts.yml

```diff
@@ -278,6 +278,10 @@ _nginx_proxy_sites:
     # Only add this CORS header to the default public site
     nginx_proxy_additional_directives:
     - "add_header Access-Control-Allow-Origin $allow_origin"
+    # Study redirects
+    - "if ($request_uri ~ /search/\\?query=Name:idr0065) {
+       return 302 /about/download.html;}"
+
```

Comment:
We already have a section for custom study redirects:
https://github.com/IDR/deployment/blob/97b70b98c64bcdc5e3bcac5ffb206e59856cb208/ansible/group_vars/proxy-hosts.yml#L206-L228
Can you add it there instead?

Reply:
That's the first thing I tried but it didn't work, I can retest. Is is possible that https://github.com/IDR/deployment/blob/master/ansible/group_vars/proxy-hosts.yml#L232 is also required ?

Reply:
Retested the following configuration on `test71`. The redirect is not working . Do you have a suggestion on how to use the redirect map for the `/search` queries?

```diff
--- a/ansible/group_vars/proxy-hosts.yml
+++ b/ansible/group_vars/proxy-hosts.yml
@@ -227,6 +227,9 @@ nginx_proxy_redirect_map:
 - match: ~/pgpc
   dest: /webclient/?show=screen-1151
 
+- match: /search/?query=Name:idr0065
+  dest: /about/download.html
+
 nginx_proxy_redirect_map_locations:
 # TODO: change to 301 when we're happy
 - location: "~ ^/(mito|tara|pgpc)($|/)"
```